### DISCUSSION TITLE
Anyone know how to correct incorrect guide data for show name?

### DISCUSSION THREAD

**Discussion by wayner9** [OpenSageTV/Community#169] [General]
There is a new version of Dora the Explorer that I am trying to record.  In the SageTV EPG the program is named DORA, but in TVDB this is called DORA (2024) and DORA is a different show.  After the show is recorded SageTV seems to pull the data from TVDB for the wrong DORA show, as in "Dora is the Croatian national selection for Croatia's Eurovision Song Contest entry."

And the metadata is screwed up and even when I try to edit it in BMT I can never edit it correctly.

**Comment by Ugadata** [comment]
I suspect I may be suggesting things You've already tried but .......

First is to uncheck the Preserve Original Metadata (if checked), then Save (you may need to refresh the page and re-search - BMT screens are not dynamic)

It may also be necessary to uncheck Sage Recording? 

Then click the Find Metadata button and fill in the search option. You will need at least the following fields:

![image](https://github.com/user-attachments/assets/64f0c14c-8018-47bb-9182-1011fcfc35cc)

then click the Search button and you should see the follwing:

![image](https://github.com/user-attachments/assets/2251ba43-07bf-45cf-9ae0-0edd16a3fb66)

Notice the Dora (2024) (highlighted in lower portion of dialog box)
Select the Dora (2024) (You can just select and click and it will search)

That should get the correct metadata for the show you want.  Be sure to Save before leaving the screen and if you want to verify whille still in BMT you will need to refresh and re-search.

Again, I know You've been around Sage for a while now and I am afraid I may be telling what you already know and have tried.

**Comment by wayner9** [comment]
Thanks for the advice - I have done much of this, but I will try some of the other things that you suggested.

**Comment by Ugadata** [comment]
Depending on what lengths you're willing to go to for correct metadata ......

Since you are dealing with actual Sage recordings, you could move the files for the series to a folder outside of what sage can "see".  I might suggest using the name of the series for the folder ( Dora (2024) ).

Rename the files  (SeriesName_S0xE0x_EpisodeName.ts - basically removing the numerical portion near the end of the filename) .  Also delete the properties files.

Then copy/move the folder into an import directory that Sage can "see"

Hopefully, this should allow Sage to detect the series correctly and/or allow the correction of any metadata not correct.

It is frustrating when Sage gets overly stubborn about changing metadata.

**Reply by wayner9** [reply]
I just went through this again and somehow it is still clinging out an old Airing ID.  I changed the filename to DORA(2024) - S01E01, etc and I deleted the number that is in the filename, moved them out of the folder, refreshed, and then brought them back, but for some reason multiple files are retaining an old Airing ID in Sage.  So several of the shows have the wrong metadata.
